### PR TITLE
try to remove nlohmann

### PR DESCRIPTION
**Pull request by yanmo96**

### CODE REVIEW

**Comment by yanmo96** [comment]
image
image
Since Travis CI is not running, I'm attaching unit test and GRPC test results as reference.

**Comment by zzxgzgz** [comment]
LGTM

**Review by xieus — APPROVED** [review]
LGTM.

File: test/gtest/aca_test_zeta_programming.cpp [hunk 1]
```diff
@@ -579,4 +586,5 @@ TEST(zeta_programming_test_cases, DISABLED_zeta_scale_container)
 
   // restore demo mode
   g_demo_mode = previous_demo_mode;
-}
\ No newline at end of file
+}
+*/
```
Comment (zzxgzgz) [inline test/gtest/aca_test_zeta_programming.cpp:590]: LGTM

**Pull request merged**